models/videos.js

```diff
@@ -0,0 +1,74 @@
+var app = require("../app");
+var db = app.get("db");
+
+// Constructor function
+var Videos = function(id) {
+  this.id = id;
+};
+
+Videos.all = function(callback) {
+  db.videos.find(function(error, videos) {
+    if(error || !videos) {
+      callback(new Error("Could not retrieve videos"), undefined);
+    } else {
+      callback(null, videos.map(function(video) {
+        return new Videos(video);
+      }));
+    }
+  });
+};
+
+Videos.find = function(title, callback) {
+  db.videos.find({title: title}, function(error, video) {
+    if (error || !video) {
+      callback(new Error("Could not retrieve video"), undefined)
+    } else {
+      callback(null, video)
+    }
+  });
+};
+
+Videos.sort = function(column, p, n, callback) {
+ db.videos.find({}, {
```

Comment:
I like how you pass the query parameters for the find method into the function as parameters. This lets you create a really flexible sorting function that can be reused for many situations.